ts/parts/Tooltip.ts

```diff
@@ -1169,7 +1192,13 @@ H.Tooltip.prototype = {
                 } as any);
             }
 
-            this.isHidden = false;
+            // show it
+            if (tooltip.isHidden && tooltip.label) {
```

Comment:
What happens if `tooltip.label` is undefined? The idea with the old structure, calling `tooltip.getLabel()` initially, is that the label is generated the first time `refresh` is called.

Reply:
`tooltip.label` should at this point always be defined, because `getLabel()` is called both in `renderSplit` and in the `else` section. The checking that it exists before use is only to satisfy TypeScript.
Previously it was called twice on split tooltip, which is the reason for this change.